src/main/java/com/zorder/orderservice/services/IProductService.java

```diff
@@ -0,0 +1,11 @@
+package com.zorder.orderservice.services;
+
+
+import com.zorder.orderservice.dto.ProductDetails;
+import com.zorder.orderservice.entities.ProductModel;
+import org.springframework.stereotype.Service;
+
+@Service
+public interface IProductService {
+    ProductModel saveinfo(ProductDetails productDetails);
```

Comment:
No need for @Service annotation, call the method saveProduct(), rather than saveInfo